packages/enhanced/src/lib/container/RemoteRuntimeModule.ts

```diff
@@ -24,7 +27,12 @@ class RemoteRuntimeModule extends RuntimeModule {
     const { runtimeTemplate, moduleGraph } = compilation as Compilation;
     const chunkToRemotesMapping: Record<string, any> = {};
     const idToExternalAndNameMapping: Record<string | number, any> = {};
-    for (const chunk of this?.chunk?.getAllAsyncChunks() || []) {
+    const allChunks = [
```

Comment:
@2heal1  this adds eager referenced to the map, so i can call chunkHandlers for entrypoints and get the requirements. like async boundary does